app/build.gradle

```diff
@@ -24,6 +24,10 @@ android {
         }
     }
 
+    buildFeatures{
+        viewBinding true;
+    }
```

Comment:
I think this should already have been in the gradle, its probably best to avoid messing with the gradle files.

Reply:
Just double checked, and it is indeed already in the gradle on main.

Reply:
Ah I see. I fix this part on my new commit.